### PR TITLE
added signup

### PR DESCRIPTION
**Pull request by sushmita12345**
Added signup page to the E-commerce website.

Sample
Screenshot 2022-02-11 at 2 14 23 AM

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
media queries break points can be standard.. such as 768px, but it is not a breaking thing. It is good to go as of now

File: index.html [hunk 1]
```diff
@@ -41,23 +40,23 @@
...
         <div class="upper-section-4">
             <img class="upper-image" src="/assets/img/model_image.jpg" />
-            <button onclick="location.href='product1.html'" target="_blank" class="upper-btn">Product</button>
+            <button onclick="location.href='/components/product1.html'" target="_blank" class="upper-btn">Product</button>
         </div>
         <div class="upper-section-5">
             <img class="upper-image" src="/assets/img/model_image.jpg" />
-            <button onclick="location.href='product2.html'" target="_blank" class="upper-btn">Product</button>
+            <button onclick="location.href='/components/product2.html'" target="_blank" class="upper-btn">Product</button>
         </div>
```
Comment (deekshithmd) [inline index.html:62]: code looks clean and good

**Review by deekshithmd — APPROVED** [review]

File: components/signup.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    <input class="login-box" type="password" value="sushmita">
+                </div>
+                
+                <div class="pwd-remember">
+                    <div class="rem-container">
+                        <input class="login-checkbox" type="checkbox">
+                        <span class="login-remember">I accept all Terms & Conditions</span>
+                    </div>
+                    
+                    <!-- <span class="login-forget">Forget your password?</span> -->
```
Comment (PuneetP16) [inline components/signup.html]: Remove unsued comments.

**Review by PuneetP16 — APPROVED** [review]
GTM.

File: components/signup.html (hunk 2 above)
Comment (sushmita12345) [reply components/signup.html]: Ok, thanks I'll update.

**Pull request merged**